.github/workflows/publish.yml

```diff
@@ -0,0 +1,20 @@
+name: Publish
+on:
+  release:
+    types:
+      - published
```

Comment:
memo) https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#release

Reply:
If we trigger on `release`, we’d have to manually create a release in the GitHub UI, right?

Another option is to run the workflow when a change to version.rb has been merged into `master` branch, like this:
```yml
on:
  push:
    branches: master
    paths: lib/rubocop/rspec/version.rb
```

Reply:
Indeed I updated this PR. Thank you .